src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const toObject = {};
```

Comment:
The variable name 'toObject' could be more descriptive. Consider renaming it to reflect that it represents style properties, such as 'styleObject' or 'cssProperties'.

Reply:
Fix it 
![Uploading image.png…]()